specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -5617,16 +5601,16 @@
         "platformFaultDomain": {
```

Comment:
is this also required property as platformFaultDomainCount?

Reply:
Not always, only if the platformDomainCount is > 1.